e2e/wdio/core/pages/dynamic-page.po.ts

```diff
@@ -0,0 +1,48 @@
+import { waitForElDisplayed, waitForPresent } from '../../driver/wdio';
+import { CoreBaseComponentPo } from './core-base-component.po';
+
+export class DynamicPagePo extends CoreBaseComponentPo {
+    private url = '/dynamic-page';
+    
+    defaultExample = 'fd-dynamic-page-basic-example ';
+    tabsExample = 'fd-dynamic-page-tabs-example ';
+    columnLayoutExample = 'fd-dynamic-page-column-layout-example ';
+    responsiveExample = 'fd-dynamic-page-responsive-example ';
+
+    dynamicPageContent = '.fd-dynamic-page__content ';
+
+    button = '.fd-button';
+    dynamicPage = '.fd-dynamic-page ';
+    acceptButton = this.button + '[label="Accept"]';
```

Comment:
better not to use label or aria-label as these can change often. try looking for unique parts of the class. for example, accept btn has:  .fd-button--positive and reject has .fd-button--negative 
try to use class as much as possible and avoid text labels